native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/common/states/Http2StateUtil.java

```diff
@@ -36,7 +36,6 @@
 import io.ballerina.stdlib.http.transport.message.Http2DataFrame;
 import io.ballerina.stdlib.http.transport.message.Http2InboundContentListener;
 import io.ballerina.stdlib.http.transport.message.Http2PushPromise;
```

Comment:
Removing this will remove the api doc reference in https://github.com/ballerina-platform/module-ballerina-http/blob/dc64c1f6e1b0708bbea032d1852a278e15fa58c1/native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/common/states/Http2StateUtil.java#L286

Reply:
Thanks @dilanSachi - I have overlooked that. Is 7fe600311c621eaa844aa3be741aa513fa6d9001 what you meant?

Reply:
Added a minor comment